### ISSUE TITLE
Sim stops working when 2 balloons deplete sweater of charges

### ISSUE THREAD

**Issue by terracoda** [phetsims/balloons-and-static-electricity#217]
When playing with two balloons, I noticed consistently (reproduced 4 times) that as soon as all charges are deplete from the sweater, one can no longer move the balloons.

One can still listen to the accessible labels and descriptions, but nothing happens in the simulation. It is completely frozen.

Adding system information:
MacBook Air Sierra 10.12.1 VO and Safari 10.0.1 (12602.2.14.0.7)

**Comment by jessegreenberg** [comment]
Thanks @terracoda I will take a look!

**Comment by jessegreenberg** [comment]
Fixed in the above commit @terracoda.  The second balloon was trying to find where more charges were when there were none left.

Here is a new version where this should be fixed:

http://www.colorado.edu/physics/phet/dev/html/balloons-and-static-electricity/1.1.6-dev.2/balloons-and-static-electricity_en.html?hideChargeControls

Sorry about that!  If it is fixed in the above version please close, otherwise assign back to me.

**Comment by terracoda** [comment]
@jessegreenberg, thanks. It works now! Closing this issue.
